docs/src/pages/components/geo/fragments/map-initial-viewport.react.mdx

```diff
@@ -0,0 +1,22 @@
+```jsx{12}
+import { AmplifyMap } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+
+import '@aws-amplify/ui-react/styles.css';
+
+import awsExports from './aws-exports';
+
+Amplify.configure(awsExports);
+
+export default function InitialViewport() {
+  return (
+    <AmplifyMap
```

Comment:
Is it too late to change the component name?  This is the only component that uses "Amplify" in the component name.

Reply:
Could we hold off on doing this, since it would be a larger refactor. We also have several Vue components that have `Amplify` in it's name, that maybe we should rename?

`"AmplifyButton",
          "AmplifyCheckBox",
          "AmplifyTextField",`

Reply:
Renamed to `MapView` in 01ca496fe81585c7fe5a88ab34a291dad66434ed